portal/views/portal.py

```diff
@@ -144,6 +144,18 @@ def exercise_and_diet():
 def lived_experience():
     return render_template('gil/lived-experience.html', user=current_user())
 
+@portal.route('/stories/david-andrew-story')
+def david_andrew_story():
```

Comment:
Actually- it's probably better to open a new PR for refactoring **all** of them to use a single view- maybe just try it out for the new views/templates ones you've added now?